src/modules/geospatial/dto/get-addresses-response.dto.ts

```diff
@@ -0,0 +1,49 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { ENUMS, GEOSPATIAL } from '@ukef/constants';
+
+export type GetAddressesResponse = GetAddressesResponseItem[];
+
+export class GetAddressesResponseItem {
+  @ApiProperty({
+    description: 'Organisation name if available',
+    example: 'CHURCHILL MUSEUM & CABINET WAR ROOMS',
```

Comment:
Could this be moved to a constant `GEOSPATIAL.EXAMPLES.ORGANISATION_NAME`? Same for the other properties